blkn/src/Input/Label.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from "react";
+import Typography from "./../Typography";
+
+type Props = {
+  label?: string,
+  isEmptyValue: boolean,
```

Comment:
`valueEmpty` is programatically neater. no useless `is`, since it's a boolean already, and yoda nomenclature makes more sense in these things IMO